packages/theme/src/components/modal.ts

```diff
@@ -90,6 +90,7 @@ function getSize(value: string): PartsStyleObject<typeof parts> {
           minH: "-webkit-fill-available",
         },
         my: 0,
+        borderRadius: "none",
```

Comment:
@RubenSmn I think it would be more proper to go with `0` instead of `none`. I tried `none` in the browser and it looks like it's not accepted.
<img width="228" alt="Screen Shot 2022-06-09 at 8 53 14 AM" src="https://user-images.githubusercontent.com/39889198/172851533-259e4335-3674-4a6d-a7e0-c0a2464ee256.png">